### DISCUSSION TITLE
How can I can get geo points on road like this ?

### DISCUSSION THREAD

**Discussion by jesussmile** [liodali/osm_flutter#252] [Ideas]
As seen in the picture below, is it possible to get the geo points of the roads from OSM? I checked the routing but it only shows a few points? If I can get the points (every few meters ) then  I can use navigation by comparing the points and rerouting if necessary if there is a deviation.

![navigation](https://user-images.githubusercontent.com/11044978/163753649-313e21a8-711b-49cb-b39b-3649e125d4cf.jpg)

**Comment by liodali** [comment]
for now that's impossible but for routing it's depend on the length and destination point 
If are too close route will contain few waypoints
you can check if their is deviation or not  by making another demande to route and match between them
